bindings/xmtp_rust_swift/.gitignore

```diff
@@ -13,4 +13,10 @@ bundle.zip
 /target
 /Cargo.lock
 .build
-./Generated
\ No newline at end of file
+
+# Ignore swift files in include/
+/include/Generated/*.swift
+/include/Generated/xmtp_rust_swift/*.swift
```

Comment:
We can't ignore the `.h` files as well?

Reply:
The .h files actually need to be added to `include` and exported along with the XCFramework. Not sure if this is the ideal way but works and avoids having us require the app developer to add separate headers as source in their XCode project. From high level it makes sense to package the headers with the static library.